SDK/MobileCenterCrashes/WatsonRegistrationUtility/WatsonRegistrationManager.cpp

```diff
@@ -9,3 +9,8 @@ void WatsonRegistrationManager::Start(Platform::String^ appSecret)
 {
 	WerRegisterCustomMetadata(TEXT("VSMCAppSecret"), const_cast<LPWSTR>(appSecret->Data()));
 }
+
+void WatsonRegistrationManager::SetCorrelationId(Platform::String^ correlationId)
+{
+	WerRegisterCustomMetadata(TEXT("VSMCCorrelationId"), const_cast<LPWSTR>(correlationId->Data()));
```

Comment:
Since `WerRegisterCustomMetadata` takes `PCWSTR` (read as `const WCHAR *`) in parameter, `const_cast` doesn't needed here.
Situation in function above is the same.

Reply:
True, but I'm probably going to leave the code as is because it was given to me by the Watson team directly